### PR TITLE
Remove DEV pattern.

### PR DESCRIPTION
**Pull request by hasinidilanka**
**Proposed Changes**

- Remove DEV pattern in new config model.
- Add the inferred values to the deployment.toml file.

Related To: https://github.com/wso2/product-is/issues/6111

**When to Merge**

- When the kernel is updated to alpha

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml [hunk 1]
```diff
@@ -17,15 +16,27 @@ connection_password = "admin"
...
+validation_query = "SELECT 1"
+username = "wso2carbon"
+password = "wso2carbon"
 
 [keystore.tls]
 name = "wso2carbon.jks"
 password = "wso2carbon"
 
+[truststore]
+name = "client-truststore.jks"
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml]: shall we add this to default file?

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_addremaininguserattribute.toml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 [server]
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_addremaininguserattribute.toml]: rename the file name "addremaininguserattribute.toml" should have a separator

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_convertToOIDC.toml [hunk 3]
```diff
@@ -1,8 +1,7 @@
 [server]
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_convertToOIDC.toml]: rename the file, word separator is not consistant

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/JdbcUserMgtConfig.toml [hunk 4]
```diff
@@ -1,8 +1,7 @@
 [server]
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/JdbcUserMgtConfig.toml]: rename file as mentioned in above comments

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/readOnlyLdapUserMgtConfig.toml [hunk 5]
```diff
@@ -1,8 +1,7 @@
 [server]
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/readOnlyLdapUserMgtConfig.toml]: rename file as mentioned in above comments

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml (hunk 1 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml]: It is added in the default file. But in production users should change this and shouldn't this config visible to the uesrs?

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_addremaininguserattribute.toml (hunk 2 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_addremaininguserattribute.toml]: Fixed.

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_convertToOIDC.toml (hunk 3 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/jwt_token_issuer_convertToOIDC.toml]: Fixed.

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/JdbcUserMgtConfig.toml (hunk 4 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/JdbcUserMgtConfig.toml]: Fixed.

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/readOnlyLdapUserMgtConfig.toml (hunk 5 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/userMgt/readOnlyLdapUserMgtConfig.toml]: Fixed.

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml (hunk 1 above)
Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/scim/IDENTITY4776/catalina_server_config.toml [hunk 6]
```diff
@@ -33,7 +32,7 @@ name = "wso2carbon.jks"
 password = "wso2carbon"
 
 [transport.https]
-properties.maxThreads = "1"
+maxThreads = "1"
 
 [transport.http]
-properties.maxThreads = "1"
\ No newline at end of file
+maxThreads = "1"
```
Comment (jeradrutnam) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/scim/IDENTITY4776/catalina_server_config.toml]: Missing newline at EOF.

Comment (hasinidilanka) [reply modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/scim/IDENTITY4776/catalina_server_config.toml]: Fixed in master

**Pull request merged**